README.md

```diff
@@ -5,7 +5,7 @@
   <img src="mathmagician.jpeg" alt="logo" width="140"  height="auto" />
   <br/>
 
-  <h3><b>MAth Magician Read Me</b></h3>
```

Comment:
- [ ] Kindly remove these boilerplate files.

  ![mjd](https://user-images.githubusercontent.com/36542986/226864501-f63ede38-6982-4f0d-9a27-b0d9e83b4194.jpg)

Reply:
## OPTIONAL 
* As this was missed by the precious CR I mark this as optional but would strongly suggest you change your components to functional components as its a requirement [Check here ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20functional%20components)